MAUI/TextInputLayout/Events.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: post
+title: Events in MAUI TextInputLayout control | Syncfusion
+description: Learn about Events support in Syncfusion Essential Studio MAUI TextInputLayout control, its elements, and more.
+platform: maui
+control: SfTextInputLayout
+documentation: ug
+---
+
+# Events in MAUI TextInputLayout
+
+## PasswordVisibilityToggled Event
+
+The `PasswordVisibilityToggled` event will be triggered whenever toggle the EnablePasswordVisibilityToggle icon in the SfTextInputLayout. It can defines visibility of the password based on the icon clicked. If the icon shows, the value of `IsPasswordVisible` will be **true**. The event arguments are of type `PasswordVisibilityToggledEventArgs` and expose the following property:
+
+* `IsPasswordVisible`: Its value is defined based on the password visibility.
```

Comment:
IsPasswordVisible: Its value is defined based on the visibility of the password.

Reply:
Addressed the comments.